lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf

```diff
@@ -250,9 +250,9 @@ emqx_ee_bridge_kafka {
     kafka_message_key {
         desc {
             en: "Template to render Kafka message key. "
-                "If the desired variable for this template is not found in the input data "
+                "If the template is rendered into an empty string, "
                 "<code>NULL</code> is used."
```

Comment:
Q: is that done somehow by `kpro` when it encounters an empty binary?  I couldn't find mentions of `null` in `wolff_producer` :thinking:

Reply:
it's done in `kafka_protocol`. empty strings are encoded as `NULL`.

Reply:
I actually need to check if it's `"undefined"` or `""` for unknown variables.